viral/two_photon.py

```
@@ -174,7 +151,41 @@ def get_position_activity(
 
         test_matrices.append(test_matrix[sorted_order, :])
 
-    return np.mean(np.array(test_matrices), 0)
+    test_matrices_averaged = np.mean(np.array(test_matrices), 0)
+
+    if ITI_bin_size is None:
+        return test_matrices_averaged
+
+    # Either return the ITI, or random chunks of rest, remove the comments
+    # When decide what to do
+    resting = get_resting_chunks(
```

Ruff (0.8.2)

Undefined name `get_resting_chunks`

(F821)